modules/cert-manager-override-flag-controller.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-flag-controller_{context}"]
+= Deleting TLS secret automatically upon Certificate removal
+
+You can enable the `--enable-certificate-owner-ref` flag for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
```

Comment:
What does the `--enable-certificate-owner-ref` flag do?

Reply:
I see the definition below. Maybe move the _ The `--enable-certificate-owner-ref` flag sets..._ sentence here? It might be more helpful earlier in the topic.

Reply:
Agree